Makefile

```diff
@@ -14,6 +15,7 @@ deps:
 .PHONY: testdeps
 testdeps:
 	go get -d -v -t ./...
```

Comment:
We don't need `-t` here right?

Reply:
In module-aware mode, yes. However I think Travis CI should run `make cover` in GOPATH mode too. This p-r makes to do it (fixed).